cpp/test/Slice/errorDetection/WarningInvalidMetadata.err

```diff
@@ -22,13 +22,13 @@ WarningInvalidMetadata.ice:31: warning: the 'cpp:no-default-include' metadata do
 WarningInvalidMetadata.ice:34: warning: 'amd' metadata cannot be specified as file metadata
 WarningInvalidMetadata.ice:40: warning: the 'cpp:header-ext' metadata only accepts one argument but a list was provided
 WarningInvalidMetadata.ice:40: warning: 'cpp:header-ext' metadata cannot be applied to sequences
+WarningInvalidMetadata.ice:44: warning: ignoring duplicate metadata: 'deprecated:do not use this' does not match previously applied metadata of 'deprecated'
```

Comment:
And finally, the original reason for doing this:
now all diagnostics are in line-number ordering.